source/ed/ed2nav.h

```diff
@@ -45,19 +45,18 @@ namespace ed {
 template <class T = navitia::type::Data>
 bool try_save_file(const std::string& filename, const T& data) {
     auto logger = log4cplus::Logger::getInstance("ed2nav::try_save_file");
+    LOG4CPLUS_INFO(logger, "Trying to save " << filename);
     try {
         data.save(filename);
     } catch (const navitia::exception& e) {
         LOG4CPLUS_ERROR(logger, "Unable to save " << filename);
         LOG4CPLUS_ERROR(logger, e.what());
         return false;
     }
+    LOG4CPLUS_INFO(logger, "File " << filename << " has been saved");
     return true;
 }
 
-bool rename_old_file(const std::string& output_filename, const std::string& backup_output_filename);
```

Comment:
minor-minor: You may add a prototype for `rename_file` however (I get a warning from that).

Reply:
What version of clang do you use ? So that I can check the warnings in my clang-tidy PRs.